missing theme/styled.d.ts file in dist

### What happened and what did you expect to happen?

I'm getting errors when running my code through `tsc`

`node_modules/amazon-chime-sdk-component-library-react/lib/theme/dark.d.ts:2:25 - error TS2307: Cannot find module './styled' or its corresponding type declarations.`

`node_modules/amazon-chime-sdk-component-library-react/lib/theme/light.d.ts:2:25 - error TS2307: Cannot find module './styled' or its corresponding type declarations.`

Then I started comparing my code to the [meeting demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/meeting). The differences I've noticed are:

- The demo is not running `tsc` on the code to perform type checking
- `skipLibCheck` is set to `true` in [tsconfig.json](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/meeting/tsconfig.json#L17)

I switch `skipLibCheck` to `false` and ran the code through `tsc` and started getting the type errors I was seeing in my app, as well as a couple of others.

See below for exact steps and console output on the meeting demo.


### Have you reviewed our existing documentation?

- [X] Amazon Chime SDK for JavaScript GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] Amazon Chime SDK React Components Library GitHub [issues](https://github.com/aws/amazon-chime-sdk-component-library-react/issues)
- [X] Storybook [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-component-library-react#readme)
- [X] How-to [documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page)
- [X] React SDK meeting [demo](https://github.com/aws/amazon-chime-sdk-component-library-react/tree/master/demo/meeting#readme---react-meeting-demo)

### Reproduction steps

1. `git clone git@github.com:aws-samples/amazon-chime-sdk.git`
2. `cd apps/meeting`
3. `npm i`
4. change `skipLibCheck` to false in `tsconfig.json`
5. `npx tsc`
6. see errors in console

### Amazon Chime SDK React Components Library version

2.8.0

### What browsers are you seeing the problem on?

N/A

### Browser version

N/A

### Device Information

macOS 11.5.2

### Meeting and Attendee ID Information.

_No response_

### Browser console logs

### Terminal Output

```
➜ npx tsc
node_modules/@types/styled-components/ts3.7/index.d.ts:166:8 - error TS2615: Type of property 'propTypes' circularly references itself in mapped type 'Pick<ForwardRefExoticComponent<Pick<Pick<any, Exclude<keyof ReactDefaultizedProps<StyledComponentInnerComponent<WithC>, ComponentPropsWithRef<StyledComponentInnerComponent<WithC>>>, StyledComponentInnerAttrs<...> | ... 1 more ... | StyledComponentInnerAttrs<...>> | Exclude<...> | Exclude<...> | Exclude<...>> & Parti...'.

166     ): StyledComponent<
           ~~~~~~~~~~~~~~~~
167         StyledComponentInnerComponent<WithC>,
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
170         A | StyledComponentInnerAttrs<WithC>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
171     >;
    ~~~~~

node_modules/@types/styled-components/ts3.7/index.d.ts:166:8 - error TS2615: Type of property 'propTypes' circularly references itself in mapped type 'Pick<ForwardRefExoticComponent<Pick<Pick<any, Exclude<keyof ReactDefaultizedProps<StyledComponentInnerComponent<WithC>, ComponentPropsWithRef<StyledComponentInnerComponent<WithC>>>, any>> & Partial<...>, any> & { ...; } & WithChildrenIfReactComponentClass<...>>, "propTypes" | ... 2 more ... | "$$typeof">'.

166     ): StyledComponent<
           ~~~~~~~~~~~~~~~~
167         StyledComponentInnerComponent<WithC>,
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
170         A | StyledComponentInnerAttrs<WithC>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
171     >;
    ~~~~~

node_modules/amazon-chime-sdk-component-library-react/lib/theme/dark.d.ts:2:25 - error TS2307: Cannot find module './styled' or its corresponding type declarations.

2     breakpoints: import("./styled").Breakpoints;
                          ~~~~~~~~~~

node_modules/amazon-chime-sdk-component-library-react/lib/theme/light.d.ts:2:25 - error TS2307: Cannot find module './styled' or its corresponding type declarations.

2     breakpoints: import("./styled").Breakpoints;
                          ~~~~~~~~~~


Found 4 errors.
```

### Add any other context about the problem here.

I was able to "fix" my errors on the missing `./styled` module by copy/pasting the file from [source](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/theme/styled.d.ts) into my node_modules here: `node_modules/amazon-chime-sdk-component-library-react/lib/theme`

I did not dig into the circular references errors I saw in the meeting demo.

--- a/src/theme/styled.ts
+++ b/src/theme/styled.ts
@@ -1,4 +1,4 @@
-// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 import 'styled-components';

--- a/src/theme/styled.ts
+++ b/src/theme/styled.ts
@@ -1,3 +1,6 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
 import 'styled-components';
 
 interface TextType {

